src/utils/generate_biolink_imports.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Dict
+from linkml_runtime import SchemaView
+from linkml.utils.schema_builder import SchemaBuilder
+from linkml_runtime.utils.formatutils import camelcase, underscore
+from linkml_runtime.dumpers import yaml_dumper
+import importlib.resources
+from linkml_runtime.linkml_model import (
+    EnumDefinition,
+    PermissibleValue)
+
+
```

Comment:
This is the main chunk of what needs a review. I slightly generalized the code to generate enumerations from slots names vs classes names (which could maybe live somewhere in linkml eventually?).